packaging/manual_nightly_deployment.sh

```diff
@@ -60,7 +60,7 @@ publish_docker() {
 	fi
 
 	echo "Building Docker images.."
-	packaging/docker/build.sh
+	RELEASE_CHANNEL=nightly packaging/docker/build.sh
```

Comment:
Maybe RELEASE_CHANNEL could default to nightly inside `packaging/docker/build.sh`

Reply:
I couldn't decide what a sane default value would be for `build.sh` so I made it mandatory to specify it before the build. Being explicit about it reduces confusion and surprises at least. We can default it to one or the other in the future without breaking any call sites.

Reply:
@knatsakis are we good to resolve this?

Reply:
@ncmans yeap, resolved